app/common_components/course_viewer/lessonCourseToggle.tsx

```diff
@@ -0,0 +1,46 @@
+interface LessonCourseToggleProps {
+	lessonToggle: boolean
+	setLessonToggle: (fn: (prev: boolean) => boolean) => void
+}
+
+const LessonCourseToggle = ({
+  lessonToggle,
+  setLessonToggle,
+}: LessonCourseToggleProps) => {
+  if (lessonToggle == true) {
```

Comment:
We can do something like this to avoid having an if-else statement that reuses a lot of the same code:

First create two `consts` at the top of the file.
```js
const selectedStyle = "text-sm py-1 px-2 rounded-tl-lg rounded-bl-lg bg-gray-300 text-black shadow-inner font-bold";
const notSelectedStyle = "text-sm py-1 px-2 rounded-tr-lg rounded-br-lg shadow-inner hover:font-bold";
```

then on the buttons use jsx and a ternary operator that checks the toggle value to determine the style"
```jsx
        <button
          className={`lessonToggle ? ${selectedStyle} : ${notSelectedStyle}`}
          onClick={() => setLessonToggle((prev) => !prev)}
        >
```
Do the same but in reverse for the second button

You should then be able to delete the else statement

Reply:
I always forget about ternary's for styling.

I've had to edit this slightly to make sure we maintain the pill shape but the code looks a lot cleaner now